securedrop/journalist_app/col.py

```diff
@@ -93,6 +95,18 @@ def download_single_file(filesystem_id: str, fn: str) -> werkzeug.Response:
         if '..' in fn or fn.startswith('/'):
             abort(404)
 
+        file = current_app.storage.path(filesystem_id, fn)
+        if not Path(file).is_file():
+            flash(
+                gettext(
+                    "An error occured: file not found. "
```

Comment:
typo: occured -> occurred

the following message incorporates most of the feedback from you and others, so if you could update the error message to the following then i think we're golden:

> Your download failed because a file could not be found. An admin can find more information in the system and monitoring logs.

(If you're curious, I mentioned more about the issue of how we want to direct the admin to the right place here: https://github.com/freedomofpress/securedrop-docs/issues/69#issuecomment-721347115)

Reply:
> typo: occured -> occurred

Dang! Can never get this one right...